# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1498059991 14400
#      Wed Jun 21 11:46:31 2017 -0400
# Node ID 3037e865dee1994b9fba57ecf046876be315f08f
# Parent  d197f62cde8a50667d6b7c27367697db86d2a81b
* ov-classdef.h: Fix typo in copyright notice.

diff --git a/libinterp/octave-value/ov-classdef.h b/libinterp/octave-value/ov-classdef.h
--- a/libinterp/octave-value/ov-classdef.h
+++ b/libinterp/octave-value/ov-classdef.h
@@ -5,17 +5,17 @@ Copyright (C) 2012-2017 Michael Goffioul
 This file is part of Octave.
 
 Octave is free software; you can redistribute it and/or modify it
 under the terms of the GNU General Public License as published by
 the Free Software Foundation; either version 3 of the License, or
 (at your option) any later version.
 
 Octave is distributed in the hope that it will be useful, but
-qWITHOUT ANY WARRANTY; without even the implied warranty of
+WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
 You should have received a copy of the GNU General Public License
 along with Octave; see the file COPYING.  If not, see
 <http://www.gnu.org/licenses/>.
 
 */
